release_notes_generator/action_inputs.py

```diff
@@ -252,3 +265,33 @@ def validate_inputs():
         logger.debug("Verbose logging: %s", verbose)
         logger.debug("Warnings: %s", warnings)
         logger.debug("Print empty chapters: %s", print_empty_chapters)
+
+    @staticmethod
+    def _detect_row_format_invalid_keywords(row_format: str, row_type: str = "Issue") -> None:
+        """
+        Detects invalid keywords in the row format.
+
+        @param row_format: The row format to be checked for invalid keywords.
+        @param row_type: The type of row format. Default is "Issue".
+        @return: None
+        """
+        keywords_in_braces = re.findall(r"\{(.*?)\}", row_format)
+        invalid_keywords = [keyword for keyword in keywords_in_braces if keyword not in SUPPORTED_ROW_FORMAT_KEYS]
+        for invalid_keyword in invalid_keywords:
+            logger.error(
+                f"Invalid `{invalid_keyword}` detected in `{row_type}` row format keyword(s) found: {', '.join(invalid_keywords)}. Will be removed from string."
```

Comment:
Do not use f string for logging use placeholders % instead. I am also not a big fan of this approach. You are using this method in validation, where you behave with errors in one way, where you add them into a list and in the final stage of the method you print them and exit the software, whenever there is at least one error. Here it is different. If you won't exit the system is it still error in this scenario or just a warning with exact behaviour?

It also smells weird to have two almost same methods. Do you have any idea for refactoring? To conclude them into one and to better handle the errors? Or am I suggesting the bad approach?

Reply:
Action merged in commit 3a77a9ec12c250ead4ad816666a634511d384a2e.

Reply:
Look way better now!:)